src/buffer/out/TextColor.cpp

```diff
@@ -90,26 +90,22 @@ COLORREF TextColor::GetColor(std::basic_string_view<COLORREF> colorTable,
 
         return defaultColor;
     }
-    else if (IsRgb())
+
+    if (IsRgb())
     {
         return _GetRGB();
     }
-    else
+
+    FAIL_FAST_IF(colorTable.size() < _index);
+    // If the color is already bright (it's in index [8,15] or it's a
+    //       256color value [16,255], then boldness does nothing.
+    if (brighten && _index < 8)
     {
-        FAIL_FAST_IF(colorTable.size() < _index);
-        // If the color is already bright (it's in index [8,15] or it's a
-        //       256color value [16,255], then boldness does nothing.
-        if (brighten && _index < 8)
-        {
-            FAIL_FAST_IF(colorTable.size() < 16);
-            FAIL_FAST_IF(gsl::narrow_cast<size_t>(_index) + 8 > colorTable.size());
-            return colorTable.at(gsl::narrow_cast<size_t>(_index) + 8);
-        }
-        else
-        {
-            return colorTable.at(_index);
-        }
+        FAIL_FAST_IF(colorTable.size() < 16);
+        FAIL_FAST_IF(gsl::narrow_cast<size_t>(_index) + 8 > colorTable.size());
+        return colorTable.at(gsl::narrow_cast<size_t>(_index) + 8);
     }
+    return colorTable.at(_index);

```

Comment:
I suppose in this PR we should do only nullptr changes, but not irrelevant ones like this. And probably simply finding and replacing all NULL to nullptr would be safe and good enough.